website/blog/2023-11-30-sharing-podman-images-with-kubernetes-cluster.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Share your local podman images with the Kubernetes cluster
+description: Build image and use it directly in your kubernetes cluster
+slug: sharing-podman-images-with-kubernetes-cluster
+authors: [benoitf]
+tags: [podman-desktop, podman, images, kubernetes]
+hide_table_of_contents: false
+image: /blog/img/sharing-podman-images-with-kubernetes-cluster/selkie-family.png
+---
+
+As developers we constantly improve and refine our applications. One of the challenges we face is quickly iterating when working with container images and kubernetes deployments/pods.
+
+For example, when we want to try a new image in a kubernetes pod, this image needs to be available on a private/public registry or on the nodes of the kubernetes cluster.
+Sometimes we need to call extra commands such as `kind load docker-image` or `minikube cache add <image>` or publish the image first to a 3rd party registry.
+
+You'll agree that trying out a new image in a Kubernetes pod should be as seamless as building the image itself.
+
+In this blog post, we will explore the best practice for streamlining the image iteration process in Kubernetes with Podman Desktop.
+
+![hero](img/sharing-podman-images-with-kubernetes-cluster/selkie-family.png)
+
+<!--truncate-->
+
+---
+
+## Introduction
+
+When using kind or minikube or other 3rd party tools to setup a local kubernetes cluster, we have several ways to publish images.
+
+Minikube published 8 ways of doing that at https://minikube.sigs.k8s.io/docs/handbook/pushing/
+
+There are pros and cons either way. Starting a registry requires publishing your image each time you want to build. Podman Desktop could automate the synchronization but there is still a duplication of layers.
```

Comment:
```suggestion
There are pros and cons either way. Using a third party registry implies that you need to publish the image after each build of the image before being able to use it in the kubernetes cluster. While Podman Desktop could automate the synchronization between the local registry (where you are doing `podman build`) and the third party registry, there remains a duplication of layers between the local and third party registry. And if you change the first layer, it can take a lot of time to send again all the data.
```


Both approaches have their advantages and disadvantages. Initiating a registry necessitates the publication of your image every time you wish to construct it. Although Podman Desktop can streamline the synchronization process, there remains a duplication of layers.

Reply:
@lstocchi better this way ?